Cargo.toml

```diff
@@ -18,7 +18,7 @@ homepage = "https://github.com/Cardinal-Cryptography/drink"
 license = "Apache-2.0"
 readme = "README.md"
 repository = "https://github.com/Cardinal-Cryptography/drink"
-version = "0.8.6"
+version = "0.8.7"
```

Comment:
we should bump to `0.9.0` as some items are missing now (like the `Runtime` trait)